src/components/oc-client/_package/src/oc-client.js

```diff
@@ -302,41 +301,16 @@ var oc = oc || {};
             callback(MESSAGES_ERRORS_LOADING_COMPILED_VIEW.replace('{0}', compiledViewInfo.src));
           } else {
             if(compiledViewInfo.type === 'handlebars'){
-              
-              /*  
-                The following code is required for rendering components
-                published with both Handlerbars V3 and V4. This polyfill
-                will be removed and kept only to allow OC mantainers to
-                gracefully migrate to the new version (supporting only v4). 
-              */
-
-              oc.require('handlebars3', HANDLEBARS3_URL, function(){
-
-                $window.handlebars3 = $window.handlebars3 || $window.Handlebars;
-                if(!!$window.Handlebars){
-                  $window.Handlebars.noConflict();
+              oc.require('Handlebars', HANDLEBARS_URL, function(){
+                try {
+                  var linked = $window.Handlebars.template(compiledView, []);
+                  callback(null, linked(model));
+                } catch(e){
+                  callback(e.toString());
                 }
-
-                oc.require('handlebars4', HANDLEBARS_URL, function(){
-
-                  $window.handlebars4 = $window.handlebars4 || $window.Handlebars;
-                  if(!!$window.Handlebars){
-                    $window.Handlebars.noConflict();
-                  }
-                  
-                  var needs3 = compiledView.compiler[0] < 7,
-                      compiler = needs3 ? $window.handlebars3 : $window.handlebars4,
-                      linked = compiler.template(compiledView, []),
-                      html = linked(model);
-
-                  callback(null, html);
-                });
               });
-
-              // End of the Handlebars "migrate" part
-
             } else if(compiledViewInfo.type === 'jade'){
-              oc.require('jade', JADE_URL, function(jade){
+              oc.require('jade', JADE_URL, function(){
```

Comment:
this is not related right? you just removed the parameter as not used by that function correct? a.k.a. "[the boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)" 😄